docs/chronicle/falcon-integration-gateway.yaml

```diff
@@ -107,11 +111,19 @@ spec:
                 valueFrom:
                   secretKeyRef:
                     name: falcon-integration-gateway-chronicle-creds
-                    key: GOOGLE_CUSTOMER_ID     
+                    key: GOOGLE_CUSTOMER_ID
+              - name: GOOGLE_SERVICE_ACCOUNT
+                value: /google-application-credentials.json      
       volumes:
           - name: config-volume
             configMap:
               name: falcon-integration-gateway-chronicle-config
               items:
               - key: config.ini
                 path: config.ini
+          - name: google-application-credentials
+            secret:
+              secretName: fdr-falcon-integration-gateway-creds
```

Comment:
I think this should fail when tried.

The secret name seems to be wrong.

Reply:
Updated, will test these changes out today.